{{cookiecutter.project_slug}}/environment.yml

```diff
@@ -20,7 +22,7 @@ dependencies:
     - pytest-cov
     - pytest-pythonpath
 {% if cookiecutter.configure_running_sphinx_from_waf == 'y' %}
-    - sphinx==1.8.5
+    - sphinx
```

Comment:
So does the Waf builder work with Sphinx 2.0+ now?

Reply:
Was a different problem than I thought, see issue #56